src/Input.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+function Input({ placeholder, value, onChange }) {
+  return (
+    <input type="text" placeholder={placeholder} value={value} onChange={(e) => onChange(e)} />
```

Comment:
```suggestion
    <input type="text" placeholder={placeholder} value={value} onChange={onChange} />
```
event의 경우 직접 넘기지 않아도 넘어갑니다~